src/pygef/cpt.py

```diff
@@ -64,6 +65,42 @@ class CPTData:
         zlm_pore_pressure_u2_after (float | None): zlm_pore_pressure_u2_after
         zlm_pore_pressure_u3_after (float | None): zlm_pore_pressure_u3_after
         data (pl.DataFrame): DataFrame
+            columns:
```

Comment:
Let's indicate which columns are always present and which are optional

Reply:
done -> https://github.com/cemsbv/pygef/pull/297/commits/607d6729b1b31da92cc0138ffdd1b5e97e00a9bd